apps/core/tests/test_event_loop_runner/test_runner_logging.py

```
@@ -99,3 +89,20 @@ def test_handle_event_exception_logs_context(self, capture_handler):
         assert event.__class__.__name__ in record.getMessage()
         assert event.uuid in record.getMessage()
         assert record.exc_info is not None
+
+    @pytest.mark.django_db
+    def test_handle_message_dispatches_command_and_event_handlers(self):
+        processed: list[str] = []
+
+        @message_registry.register_command(command=_DummyCommand)
+        def _recording_command_handler(context: _DummyContext):
```

Ruff (0.15.5)

[warning] Missing return type annotation for private function `_recording_command_handler`

(ANN202)